multiraft/multiraft_test.go

```diff
@@ -109,7 +109,10 @@ func (c *testCluster) createGroup(groupID roachpb.RangeID, firstNode, numReplica
 		c.groups[groupID] = append(c.groups[groupID], nodeIndex)
 	}
 	for i := 0; i < numReplicas; i++ {
-		gs := c.storages[firstNode+i].GroupStorage(groupID)
+		gs, err := c.storages[firstNode+i].GroupStorage(groupID, 0)
```

Comment:
This is actually using zero as an arbitrary *valid* replica ID. The MemoryStorage used in this test doesn't do anything with replica IDs.